include/exadg/compressible_navier_stokes/user_interface/parameters.h

```diff
@@ -192,6 +193,18 @@ class Parameters
   // use combined operator for viscous term and convective term in order to improve run
   // time
   bool use_combined_operator;
+
+  /**************************************************************************************/
+  /*                                                                                    */
+  /*                            ELEMENTWISE INVERSE MASS                                */
+  /*                                                                                    */
+  /**************************************************************************************/
+  // Used when matrix-free inverse mass operator is not available and when the spatial
+  // discretization is DG, e.g. simplex.
+
+  bool solve_elementwise_mass_system_matrix_free;
+
+  SolverData solver_data_elementwise_inverse_mass;
```

Comment:
Same as for ConvDiff module, see comment https://github.com/exadg/exadg/pull/380/files#r1176622661